src/onemod/config/model_config/rover_config.py

```diff
@@ -2,24 +2,52 @@
 
 FIXME: improve top_pct_score and top_pct_learner descriptions
 
+TODO: Update docstrings, default is None, if None, must inherit from
```

Comment:
Does this docstring update require a separate ticket, or perhaps part of the documentation tasks?

Reply:
Oh, that's old! I'll remove it